src/interfaces/Config/ICloudinaryConfigurations.ts

```diff
@@ -0,0 +1,12 @@
+import IAccountConfig from "./IAccountConfig";
+import ICloudinaryAssetConfigurations from "./ICloudinaryAssetConfigurations";
+import ITagConfig from "./ITagConfig";
+import IApiConfig from "./IApiConfig";
+
+interface ICloudinaryConfigurations extends ICloudinaryAssetConfigurations{
+  account?: IAccountConfig
```

Comment:
We said we will have this totally separated from the rest

Reply:
I did not know that this is what was agreed.
If we're sure we can remove the AccountAPI code from this PR.